ting_infomedia.module

```diff
@@ -27,8 +27,19 @@ function ting_infomedia_menu() {
  * Return a render array
  */
 function ting_infomedia_view($id) {
+  global $user;
+  if( !ding_user_is_provider_user($user)){
```

Comment:
OK. Then I think it would be a good thing to add a inline comment explaining this.